libs/ui-components/src/components/EnrollmentRequest/EnrollmentRequestList.tsx

```
@@ -48,10 +48,15 @@ type EnrollmentRequestListProps = {
   isStandalone?: boolean;
 };
 
+const enrollmentRequestListPermissions = [
+  { kind: RESOURCE.ENROLLMENT_REQUEST_APPROVAL, verb: VERB.POST },
+  { kind: RESOURCE.ENROLLMENT_REQUEST, verb: VERB.DELETE },
+];
+
 const EnrollmentRequestList = ({ onEmptyListChanged, refetchDevices, isStandalone }: EnrollmentRequestListProps) => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'EnrollmentRequestList'. Consider to delete it or rename it.

'EnrollmentRequestList' is defined here:


(lint/suspicious/noRedeclare)